src/GraphQL.Tests/Bugs/BubbleUpTheNullToNextNullable.cs

```diff
@@ -111,9 +111,15 @@ public void ListOfNonNull_containing_null_should_bubble_up_the_null()
             var errors = new[]
             {
                 new ExecutionError("Error trying to resolve field 'listOfNonNullable'.", new InvalidOperationException(
-                    "Cannot return a null member within a non-null list for list index 1."))
+                    "Cannot return null for non-null type. Field: listOfNonNullable, Type: String!."))
```

Comment:
for **a** non-null type ?

Reply:
I agree.  (Personally I think this whole message could be revised.)

Reply:
Done